src/typings/tacoPackageLoader.d.ts

```diff
@@ -27,6 +28,17 @@ declare module TacoUtility {
         public static mockForTests: TacoUtility.ITacoPackageLoader;
 
         /**
+         * Returns a path to the specified command exported from the specified package. If the package is not already downloaded,
+         * then first download and cache it locally.
+         *
+         * @param {string} packageName The name of the package to load
+         * @param {string} packageVersion The version of the package to load. Either a version number such that "npm install package@version" works, or a git url to clone
+         * @param {string} commandName The name of the binary to find
+         * @returns {Q.Promise<string>} A promise which is either rejected with a failure to find the local the binary or resolved with a path to the binary
+         */
```

Comment:
I think we usually put these headers with the implementation as well (in the *.ts file).

Otherwise I sign off on the PR!